ft_statistics_montecarlo.m

```diff
@@ -77,6 +77,7 @@
 %   cfg.voxelstatistic   deprecated
 %   cfg.voxelthreshold   deprecated
 %   cfg.precondition     before|after|[], for the statfun
+%   cfg.precision        single|double (default)
```

Comment:
I understand why you would want this, but are you sure that this is going to work, i.e. will the typecasting be done correctly if cfg.precision is 'single' and the data is 'double'. Also, why would it only be used as an argument in this function, and not in other functions with a potentially large memory footprint. 
I think in general, also, it would be good not to put commits with two different purposes into a single PR for transparency.

Reply:
I agree with you, that it is not ideal to pack different things into one PR, but I did not want to submit another PR just for this minor change. 
I agree with you also in that other functions may also benefit from setting precision. but this was the only function I had an issue with even on our cluster with a generous amount of RAM. Considering temporal data combined with a large number of permutations, this function in no doubt consumes the most memory. Besides, going through the whole codebase is beyond my paycheck. :)
Regarding functionality, yes, it works as expected at least for my datasets.